src/02-common-principles.md

```diff
@@ -71,7 +72,33 @@ not covered by this standard. The most sensible place to put it is next to the
 continuous recording file with the same naming scheme but different extensions.
 The solutions will change from case to case and publicly available datasets will
 be reviewed to include common data types in the future releases of the BIDS
-spec.
+specification.
+
+## File name structure
+
+File names consist of chains of *entities*, or key-value pairs, a *suffix* and an
```

Comment:
We were switching between plural and singular, and I think it works better to go consistently singular than consistently plural.

```suggestion
A file name consists of a chain of *entities*, or key-value pairs, a *suffix* and an
```